application/models/user_access_token_test.go

```diff
@@ -520,7 +524,95 @@ func (ms *ModelSuite) TestUserAccessToken_Renew() {
 		t.Run(test.name, func(t *testing.T) {
 			u := test.token
 			_ = u.Renew()
-			ms.False(u.IsExpired())
+			ms.False(u.DeleteIfExpired())
 		})
 	}
 }
+
+func createFixtures_UserAccessTokensDeleteExpired(ms *ModelSuite, t *testing.T) AccessTokenFixtures {
+	ResetTables(t, ms.DB)
+
+	org := Organization{Uuid: domain.GetUuid(), AuthConfig: "{}"}
+	createFixture(t, &org)
+
+	users := Users{
+		{Email: t.Name() + "_user1@example.com", Nickname: t.Name() + " User1", Uuid: domain.GetUuid()},
+		{Email: t.Name() + "_user2@example.com", Nickname: t.Name() + " User2", Uuid: domain.GetUuid()},
+	}
+	for i := range users {
+		createFixture(t, &(users[i]))
+	}
+
+	userOrgs := UserOrganizations{
+		{OrganizationID: org.ID, UserID: users[0].ID, AuthID: users[0].Email, AuthEmail: users[0].Email},
+		{OrganizationID: org.ID, UserID: users[1].ID, AuthID: users[1].Email, AuthEmail: users[1].Email},
+	}
+	for i := range userOrgs {
+		createFixture(t, &(userOrgs[i]))
+	}
+
+	tokens := UserAccessTokens{
+		{
+			UserID:             users[0].ID,
+			UserOrganizationID: userOrgs[0].ID,
+			AccessToken:        HashClientIdAccessToken("abc123"),
+			ExpiresAt:          time.Unix(0, 0),
+		},
+		{
+			UserID:             users[1].ID,
+			UserOrganizationID: userOrgs[1].ID,
+			AccessToken:        HashClientIdAccessToken("xyz789"),
+			ExpiresAt:          time.Date(2099, time.December, 31, 0, 0, 0, 0, time.UTC),
+		},
+	}
+	for i := range tokens {
+		createFixture(t, &(tokens[i]))
+	}
+
+	return AccessTokenFixtures{
+		Users:            users,
+		UserAccessTokens: tokens,
+	}
+}
+
+func (ms *ModelSuite) TestUserAccessToken_UserAccessTokensDeleteExpired() {
+	t := ms.T()
+	ResetTables(t, ms.DB)
+
+	f := createFixtures_UserAccessTokensDeleteExpired(ms, t)
+
+	got, err := UserAccessTokensDeleteExpired()
+
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
+		return
+	}
+
+	want := 1
+
+	if got != want {
+		t.Errorf("Wrong count of deleted tokens. Expected %v, but got %v", want, got)
+		return
+	}
+
+	var uats UserAccessTokens
+	if err := DB.All(&uats); err != nil {
+		t.Errorf("Couldn't get tokens in order to complete test ... %v", err)
+		return
+	}
+
+	got = len(uats)
+
+	if got != want {
+		t.Errorf("Deleted wrong number of tokens. Expected %v, but got %v", want, got)
+		return
+	}
+
+	gotToken := uats[0].AccessToken
+	wantToken := f.UserAccessTokens[1].AccessToken
+
+	if gotToken != wantToken {
+		t.Errorf("Wrong token remaining. \nWant %v \n Got %v", wantToken, gotToken)
+	}
+
+}
```

Comment:
suggest using `ms.Equal` for these comparisons

Reply:
Good, thanks.